### ISSUE TITLE
The architecture of tegola for developer

### ISSUE THREAD

**Issue by exotfboy** [terranodo/tegola#279]
Hi:

I just found tegola which is a wonderful tool for vector tile serving. 

I want to make some changes , however I can not  make it clear enough about how tegola work even I read the source code (I am new for golang too :( ). For example, the architecture if this project, the work procedure and etc.
Maybe a contribution guide like mapbox-js ?

**Comment by ARolek** [comment]
@exotfboy good call! We have been intending to write contributing guidelines for a while. We're close to releasing v0.6.0 of tegola at which point documentation is going to get some attention. I will add this task to the list. In the mean time, there are some quick instructions on how to build from source at: https://github.com/terranodo/tegola#building-from-source

**Comment by exotfboy** [comment]
Glad to hear that. 👍
